run-ut.sh

```diff
@@ -24,6 +24,7 @@ ROOT=`cd "$ROOT"; pwd`
 export DORIS_HOME=${ROOT}
 
 . ${DORIS_HOME}/env.sh
+export BUILD_TYPE=DEBUG
```

Comment:
Why using this type?

Reply:
when runing unit test we need to enable DECHEK() to found more problems